src/parser/jobs/drg/modules/BattleLitany.tsx

```diff
@@ -46,209 +97,123 @@ class BLWindow {
 // the character selected for analysis. windows that clip into or overwrite other
 // DRG litanies will be marked.
 // Used DNC technical step as basis for this module.
-export default class BattleLitany extends Analyser {
+export class BattleLitany extends BuffWindow {
```

Comment:
Comment on this class needs cleaned up - legacy comment information from before the BuffWindow conversion.  (I tried to highlight the comment but GitHub was mean because nothing in the comment lines changed)

Reply:
ah yeah, thanks for the heads up, have updated that comment